components/builder-depot/src/server.rs

```diff
@@ -959,6 +965,9 @@ fn package_channels(req: &mut Request) -> IronResult<Response> {
         }
         request.set_ident(ident);
     }
+    if session_id.is_some() {
+        request.set_account_id(session_id.unwrap());
```

Comment:
I wonder if we might want to add this functionality to the router, so it can be supported for all calls, instead of setting on a per-message basis.  That way, we could pass in an optional account id to the route_message, and the recipient could use it as and when it was needed.

Reply:
That sounds nice, but I think maybe it's worth saving that idea for a future refactor.